.github/workflows/pr-docs.yaml

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: Galasa documentation build
+
+on:
+  workflow_call:
+    inputs:
+      docs-changed:
+        description: 'True if this module has been changed and should be rebuilt'
+        required: true
+        type: string
+      artifact-id:
+        description: 'The Workflow Run ID of the last workflow containing artifacts to download'
+        required: true
+        type: string
+
+jobs:
+  log-unchanged:
+    name: Docs are unchanged
+    if: ${{ inputs.docs-changed == 'false' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Log this module is unchanged
+        run: |
+          echo "The documentation source is unchanged"
+        
+  build-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Could this `checkout` action resemble the one in the Main build please, and also have a new line inbetween that and the next step, or its quite hard to read